code/modules/chemistry/tools/dispensers.dm

```diff
@@ -602,3 +598,49 @@
 		else
 			src.icon_state = initial(src.icon_state)
 
+/obj/item/reagent_containers/food/drinks/chemtank
+	name = "unlabeled chemical barrel"
+	desc = "For storing medical chemicals and less savory things. It can be labeled with a pen."
+	inhand_image_icon = 'icons/mob/inhand/hand_general.dmi'
```

Comment:
general convention for 2h item_states is a single `statename-LR` instead of having `statename-L` and `statename-R`